h2o-genmodel/src/main/java/hex/genmodel/algos/rulefit/RuleFitMojoReader.java

```diff
@@ -0,0 +1,110 @@
+package hex.genmodel.algos.rulefit;
+
+import hex.genmodel.MultiModelMojoReader;
+import java.io.IOException;
+
+public class RuleFitMojoReader extends MultiModelMojoReader<RuleFitMojoModel> {
+
+  @Override
+  protected void readParentModelData() throws IOException {
+    _model._linearModel = getModel((String) readkv("linear_model"));
+    _model.model_type = readkv("model_type");
```

Comment:
Since this keys are in *MojoReader and *MojoWriter I would choose some external class with constants.

I know that it is too much to ask and it is a style of all MOJOs and out of scope of this PR.